packages/plugins/blocks/blocks-antd/src/blocks/TextInput/schema.json

```diff
@@ -9,6 +9,12 @@
         "default": false,
         "description": "Allow the user to clear their input."
       },
+      "type": {
+        "type": "string",
+        "enum": ["text", "number", "password", "tel", "email", "url"],
```

Comment:
We also have a `PasswordInput` block. Does it make sense to set type `password` on a normal text input? Same with number I guess?

Reply:
I think it does. Yes, there are purpose-built blocks, but this is a standard html implementation of <input type=".." /> which leaves it to the dev to decide how to use or not in edge cases like optimising for mobile etc.